packages/hooks/src/useStatelessReducer/useStatelessReducer.mdx

```diff
@@ -0,0 +1,75 @@
+---
+name: useStatelessReducer
+menu: Hooks
+route: /hooks/use-stateless-reducer
+---
+
+import { Playground } from "docz";
+import { useState } from "react";
+import { useStatelessReducer } from "@jobber/hooks";
+import { Form } from "@jobber/components/Form";
```

Comment:
`Form` isn't used in this mdx. Can be removed

Reply:
resolved in fc41fb0